prometheus-metrics-ghc/src/Prometheus/Metric/GHC.hs

```diff
@@ -124,14 +128,61 @@ ghcCollectors = [
             CounterType
             parMaxBytesCopied
     ]
+#else
+ghcCollectors :: [IO [SampleGroup]]
+ghcCollectors = [
+      statsCollector "gcs" "Total number of GCs" CounterType gcs
```

Comment:
A couple things:
  * Let's prefix these with `ghc_` to stay consistent with the existing metrics.
  * Since we are starting fresh here, let's follow [best practices](https://prometheus.io/docs/practices/naming/#base-units) and use seconds for measuring time.
  * Please line wrap these.